upload_assets.sh

```diff
@@ -7,10 +7,14 @@ CACHE_CONTROL="public,max-age=31536000"
 S3_DIR=s3://mbta-dotcom/$APP
 BUILD_TAG=${1}
 TEMP_DIR=$(mktemp -d)
-STATIC_DIR=$TEMP_DIR/priv/static
+STATIC_DIR=$TEMP_DIR/static
 
 pushd "$TEMP_DIR" > /dev/null
-sh -c "docker run --rm ${BUILD_TAG} tar -c /home/skate/priv/static" | tar -x --strip-components 2
+
+CONTAINER_ID=$(docker create $BUILD_TAG)
+docker cp $CONTAINER_ID:/home/skate/priv/static - | tar -x
```

Comment:
**question**: is there a benefit to using a pipe here?

would the following work?
```suggestion
docker cp $CONTAINER_ID:/home/skate/priv/static $STATIC_DIR
```

Reply:
Oh, good point (though because of the way it handles path stuff I think `$TEMP_DIR` is actually the correct destination since the `/static` part of the path will be preserved).

Reply:
Oh huh I guess I ran into a race condition where I didn't see your edit until I submitted my own comment, but I think your edit is correct.